dockerfiles/Dockerfile

```diff
@@ -1,16 +1,43 @@
 # syntax=docker/dockerfile:1.3-labs
 FROM ubuntu:22.04
+ARG CONTRIB_BUILD_DIR="/contrib-build"
+ARG OPENMS_VERSION=3.1.0
 
-# Step 1: set up a sane build system
-USER root
+# install build dependencies
+RUN apt-get -y update \
+  && apt-get install -y --no-install-recommends --no-install-suggests \
+    libsvm-dev \
+    g++ \
+    autoconf \
+    automake \
+    patch \
+    libtool \
+    make \
+    git \
+    gpg \
+    wget \
+    ca-certificates \
+    curl \
```

Comment:
Many of those are just necessary when using the actual contrib build mechanisms.
For building openms later on, I think you just need a compiler, cmake and a make generator (make,ninja,..)
Maybe a way to download things is also safe to keep (wget or curl)

Reply:
Good catch, I'll see what I can trim. I don't think we even need curl or wget if we install cmake with the github release install script instead of from the kitware PPA.. not that those are heavy but I think it's good to keep dependencies to the bare minimum.

I'll experiment in https://github.com/OpenMS/OpenMS/pull/7303

Reply:
You're totally right, we just needed the following packages:
- g++
- make
- git
- ca-certificates

And cmake..